Update EDW and Tensorization doc, and fix broken links (#141)
aguirre-lab/ml4c3-wiki: Tensorization.md

![ICU tensorization pipeline diagram](images/workflow/icu-tensorize.png)
> This process requires mounting our `ml4c3` directory on `mad3`, and the `bedmaster` directory on `PHS-RISC-LM4`.

### ADT table
ADT stands for Admission Discharge and Transfer. 
This table contains a time-series of all the movments in the hospital of the patients associated to the cohort. 
An ADT table lists patients by MRN, their encounter IDs, and the departments they were in. In our pipeline and data analysis, we use ADT tables to define cohorts and identify Bedmaster data.

### xref table
xref stands for Cross Reference table. This table identifies the Bedmaster files that contains data for each patient in the cohort defined by the ADT table.

> A xref table list patients by MRN, their encounter IDs, and the Bedmaster files associated to their stay with additional metadata.

Reviewer: ```suggestion
The xref table lists patients by MRN, their encounter IDs, the Bedmaster files associated with their stay, as well as additional metadata.
```
